platform/android/java/editor/build.gradle

```diff
@@ -76,26 +68,106 @@ android {
+        }
+
+        google {
+            dimension "xrRuntime"
+        }
+        meta {
+            dimension "xrRuntime"
+            applicationIdSuffix ".meta"
+            versionNameSuffix "-meta"
+        }
+        pico {
+            dimension "xrRuntime"
+            applicationIdSuffix ".pico"
+            versionNameSuffix "-pico"
+        }
+    }
+
     packagingOptions {
         // 'doNotStrip' is enabled for development within Android Studio
         if (shouldNotStrip()) {
             doNotStrip '**/*.so'
         }
     }
 
-    // Disable 'release' buildtype.
-    // The editor can't be used with target=release only, as debugging tools are then not
-    // included, and it would crash on errors instead of reporting them.
     variantFilter { variant ->
+        // Disable 'release' buildtype.
+        // The editor can't be used with target=release only, as debugging tools are then not
+        // included, and it would crash on errors instead of reporting them.
         if (variant.buildType.name == "release") {
             setIgnore(true)
         }
+
+        // Disable invalid flavors combinations
+        def names = variant.flavors*.name
+        if (names.contains("pancake")) {
+            if (names.contains("meta") || names.contains("pico")) {
+                setIgnore(true)
+            }
+        }
+        if (names.contains("openxr") && names.contains("google")) {
+            setIgnore(true)
+        }
     }
 
     applicationVariants.all { variant ->
         variant.outputs.all { output ->
-            def suffix = variant.name == "dev" ? "_dev" : ""
+            def suffix = ""
+            switch (variant.name) {
+                case "openxrMetaDev":
+                    suffix = "_openxr_meta_dev"
+                    break
+                case "openxrMetaDebug":
+                    suffix = "_openxr_meta"
+                    break
+                case "openxrPicoDev":
+                    suffix = "_openxr_pico_dev"
+                    break
+                case "openxrPicoDebug":
+                    suffix = "_openxr_pico"
+                    break
+                case "pancakeGoogleDev":
+                    suffix = "_dev"
+                    break
+                case "pancakeGoogleDebug":
+                default:
+                    suffix = ""
+                    break
+            }
             output.outputFileName = "android_editor${suffix}.apk"
         }
     }
 }
+
+configurations {
+    openxrMetaImplementation {}
+    openxrPicoImplementation {}
+}
+
+dependencies {
+    implementation libraries.kotlinStdLib
+    implementation libraries.androidxFragment
+    implementation project(":lib")
+
+    implementation "androidx.window:window:1.0.0"
+
+    // Meta OpenXR dependencies
+    openxrMetaImplementation "io.github.m4gr3d:godot-openxr-loaders-meta:1.0.0"
+
+    // Pico OpenXR dependencies
+    openxrPicoImplementation "io.github.m4gr3d:godot-openxr-loaders-pico:1.0.0"
```

Comment:
@BastiaanOlij See https://github.com/GodotVR/godot_openxr_loaders/pull/18 for the logic to enable pushing the libraries to mavencentral